src/index.js

```diff
@@ -53,7 +51,7 @@ class Game extends React.Component {
         'word': this.state.inputLetters, 
         'result': result
       };
-      this.state.count++;
+      this.setState({count: this.count++});
```

Comment:
setState is called to modify a state vatiable. IE: something in this.state.<variable> format. Since you have put count outside this.state, you need not call setState.

Reply:
1. option # 1: 
 **_count = 1
 this.count ++_**

2. option # 2:
**_this.state {
 'count' : 1
}
this.setState({count: this.state.count++});_**

Reply:
you shouldn't modify state variables directly, which was done previously. Let me try option #2.

Reply:
![screenshot from 2018-04-26 11-16-56](https://user-images.githubusercontent.com/16524546/39287885-5eb184ae-4943-11e8-9e48-81592e8aaf77.png)

with option#2, we can try option#1 if the count is not needed as a state variable

Reply:
try *this.setState({count: this.state.count+1});*